config/environments/test.rb

```diff
@@ -57,4 +57,7 @@
 
   # Enable automatic creation/migration of the test DB when running tests
   config.active_record.maintain_test_schema = true
+
+  # Maintain standard order of running tests in case of leaking changes
+  config.active_support.test_order = :sorted
```

Comment:
Yeah that makes sense, I thought they were running in random order using the `--seed` param until now